src/StackExchange.Redis/Interfaces/IServer.cs

```diff
@@ -766,10 +782,28 @@ public partial interface IServer : IRedis
         /// for the given service name.
         /// </summary>
         /// <param name="serviceName">the sentinel service name</param>
-        /// <param name="flags"></param>
+        /// <param name="flags">The command flags to use.</param>
         /// <returns>a list of the sentinel ips and ports</returns>
         Task<EndPoint[]> SentinelGetSentinelAddressesAsync(string serviceName, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Returns the ip and port numbers of all known Sentinel slaves
+        /// for the given service name.
+        /// </summary>
+        /// <param name="serviceName">the sentinel service name</param>
+        /// <param name="flags">The command flags to use.</param>
+        /// <returns>a list of the slave ips and ports</returns>
+        EndPoint[] SentinelGetSlaveAddresses(string serviceName, CommandFlags flags = CommandFlags.None);
```

Comment:
Shouldn't this be `SentinelGetSlaveAddressesByName`? Can't find previous comments, but analog seems to be those methods.

Reply:
These were existing methods:
- SentinelGetMasterAddressByName = SENTINEL get-master-addr-by-name mymaster
- SentinelGetSentinelAddresses = SENTINEL sentinels mymaster

I just added this:
- SentinelGetSlaveAddresses = SENTINEL slaves mymaster

It seems like it is following the existing `SentinelGetSentinelAddresses` method to me, but I'm happy to change it.